### PR TITLE
Added homepage link to the search results, solves #261

### PR DESCRIPTION
**Pull request by paulohp**
That's my proposal:
capture

### CODE REVIEW

**Comment by julmot** [comment]
Reference to my original issue: https://github.com/bower/bower.github.io/issues/261

**Comment by paulohp** [comment]
I fixed the security issue.

Working on more examples to show the link.

Thank you.

File: search/index.html [hunk 1]
```diff
@@ -33,8 +33,16 @@
         <tr>
           <td class="name">
             <h4>
-              <a href="{{url}}">{{name}}</a>
+              <a href="{{url}}" target="_blank" rel="noopener">{{name}}</a>
```
Comment (riyadhalnur) [inline search/index.html]: `rel="noopener noreferrer"` since FF doesn't support `noopener`

Comment (riyadhalnur) [inline search/index.html]: `rel="noopener noreferrer"` since FF doesn't support `noopener`

**Comment by sheerun** [comment]
commited on master, thank you!

**Comment by julmot** [comment]
@sheerun What about my review?

**Comment by sheerun** [comment]
@julmot I applied your suggestions before merging

**Comment by sheerun** [comment]
(except not showing full url, I missed it...)

**Pull request merged**